src/control/server/ctl_firmware_test.go

```diff
@@ -826,7 +826,7 @@ func TestCtlSvc_FirmwareUpdate(t *testing.T) {
 				runner := ioserver.NewTestRunner(&ioserver.TestRunnerConfig{
 					Running: atm.NewBool(tc.serversRunning),
 				}, ioserver.NewConfig())
-				instance := NewIOServerInstance(log, nil, nil, nil, runner)
+				instance := NewIOServerInstance(log, nil, nil, nil, nil, nil, false, runner)
```

Comment:
Starting to get the to point where we might want to pass a struct instead of adding more params

Reply:
agreed but wasn't sure whether to do that as part of this PR

Reply:
@mjmac want me to create a config struct for new IO server instances?

Reply:
Yeah I think it's time to do that. It's kind of gotten out of hand... ;)